modules/porous_flow/doc/content/source/materials/PorousFlowPermeabilityKozenyCarman.md

```diff
@@ -15,9 +15,9 @@ Input can be entered in one of two forms depending on the value of [!param](/Mat
 | --- | --- |
 | `kozeny_carman_fd2` | $A = f d^2$ |
 | `kozeny_carman_phi0` | $A = k_0 (1 - \phi_0)^m / \phi_0^n$  |
+| `kozeny_carman_A` | $A = A$  |
 
-In `kozeny_carman_fd2`, $f=$[!param](/Materials/PorousFlowPermeabilityKozenyCarman/f) is a scalar constant and $d=$[!param](/Materials/PorousFlowPermeabilityKozenyCarman/d) is the grain diameter.  In `kozeny_carman_phi0`, $k_0=$[!param](/Materials/PorousFlowPermeabilityKozenyCarman/k0) is the reference permeability and $\phi_0=$[!param](/Materials/PorousFlowPermeabilityKozenyCarman/phi0) is a reference porosity.  These input parameters are converted to $A$ internally.  For problems where $A$ is described by a spatially varying variable, use `[PorousFlowPermeabilityKozenyCarmanFromVar.md] .
-
+In `kozeny_carman_fd2`, $f=$[!param](/Materials/PorousFlowPermeabilityKozenyCarman/f) is a scalar constant and $d=$[!param](/Materials/PorousFlowPermeabilityKozenyCarman/d) is the grain diameter.  In `kozeny_carman_phi0`, $k_0=$[!param](/Materials/PorousFlowPermeabilityKozenyCarman/k0) is the reference permeability and $\phi_0=$[!param](/Materials/PorousFlowPermeabilityKozenyCarman/phi0) is a reference porosity.  These input parameters are converted to $A$ internally.  In `kozeny_carman_A`, $A=$[!param](/Materials/PorousFlowPermeabilityKozenyCarman/A) is used to provide the permeability multilpying factor directly which is useful for 2D fracture flow described [here](multiapp_fracture_flow_PorousFlow_3D.md) where A is a constant given by $A=r/12$ with $r$ being the fracture roughness. For problems where $A$ is described by a spatially varying variable, use `[PorousFlowPermeabilityKozenyCarmanFromVar.md].
```

Comment:
My only comment is to ask whether we should add a note that A > 0 when specifying it manually?

Reply:
Thanks @cpgr   I fixed the documentation and added a check in the variable version to make sure A>0.